.github/workflows/build.yml

```diff
@@ -72,6 +72,9 @@ jobs:
         path: ~/.m2
         key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
         restore-keys: ${{ runner.os }}-m2
+    - name: Build jp24u with Maven
```

Comment:
I don't think we need this extra build, since the "Build with Maven" step at line 78 will also build jp24u since its a submodule